cmd/wallet-web/src/pages/Signup.vue

```diff
@@ -79,14 +79,17 @@
                   {{ t('Signup.heading') }}
                 </h1>
               </div>
-              <div class="grid grid-cols-1 gap-5 w-full h-64 mb-8 content-center">
+              <div
+                class="grid grid-cols-1 gap-5 w-full h-64 mb-8 content-center justify-items-center"
```

Comment:
`justify-items-center` is not a tailwind class

Reply:
https://tailwindcss.com/docs/justify-items it is tailwind class, may be version is the issue